mollie-payments-for-woocommerce/includes/mollie/wc/gateway/applepay.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+use Mollie\Api\Resources\Payment;
+use Mollie\Api\Types\PaymentMethod;
+
+/**
+ * Class applepay
+ */
+class Mollie_WC_Gateway_Applepay extends Mollie_WC_Gateway_Abstract
+{
+    /**
+     * Mollie_WC_Gateway_Applepay constructor.
+     */
+    public function __construct()
+    {
+        // TODO Check if its support refund?
+        $this->supports = [
+            'products',
+            'refunds',
+        ];
+
+        parent::__construct();
+    }
+
+    /**
+     * @return string
+     */
+    public function getMollieMethodId()
+    {
+        return PaymentMethod::APPLEPAY;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDefaultTitle()
+    {
+        return __('Apple Pay', 'mollie-payments-for-woocommerce');
```

Comment:
Why does the title need to be translated here, but not the description? I would say, translate none of those things in this class. I18n should happen in the context of output, i.e. where it is necessary.

Reply:
Or i18n-ize everything.